.github/workflows/ci-actions-incremental.yml

```diff
@@ -250,21 +250,21 @@ jobs:
             name: "11",
             java-version: 11,
             maven_args: "$JVM_TEST_MAVEN_ARGS",
-            maven_opts: "-Xmx2g -XX:MaxMetaspaceSize=1g",
+            maven_opts: "-Xmx2g -XX:MaxMetaspaceSize=1300m",
```

Comment:
If just updating REST Assured required that, there might be an issue with it?

Reply:
Yeah, it seems so. I had a brief look the other day at the rest-assured and groovy issue trackers but found nothing so far.

Reply:
This looks interesting: https://issues.apache.org/jira/browse/GROOVY-10307

Reply:
WOW